requirements/common.in

```diff
@@ -52,7 +52,12 @@ django-auth-ldap
 django-bitfield
 
 # Needed for Android push notifications
-python-gcm
+# grpcio-status needs to be pinned to 1.62.* because higher
+# versions demand protobuf>5.0, which conflicts with various
+# other big dependencies of firebase-admin. pip fails to
+# find this resolution by itself.
+grpcio-status==1.62.*
+firebase-admin
```

Comment:
The dependency conflict was pretty painful to resolve :neutral_face: Flagging this here in case I'm doing something dumb and there's  a better way